sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -200,10 +229,11 @@ def batch_event_handler(event):
                     "123456789012.dkr.ecr.us-west-2.amazonaws.com/" "codice-repo:latest"
                 ),
                 "command": [
-                    "python",
                     (
-                        "imap_cli --instrument codice --level l1a "
-                        "--filename '<s3-filepath>'"
+                        "--instrument hit --level l1a "
+                        "--s3_uri 's3://bucket_name/imap/hit/l1a/2024/01/"
```

Comment:
Can we avoid `s3_uri`? I think it'd be nice if we only use the paths and let the bucket be a detail. I think that might also simplify your path parsing below too.

Reply:
I wondered that myself but Batch command was passing `s3_uri` and I changed others to match that. It gonna need some refactoring. I will revisit that demo. We will need to change in most places.